tsdate/core.py

```diff
@@ -1410,9 +1501,11 @@ def variational_dates(
         update matches mean and variance rather than expected gamma sufficient
         statistics. Faster with a similar accuracy, but does not exactly minimize
         Kullback-Leibler divergence. Default: False.
-    :param bool global_prior: If `True`, an iid prior is used for all nodes,
-        and is constructed by averaging gamma sufficient statistics over the free
-        nodes in `priors`. Default: True.
+    :param int global_prior: The number of components in the i.i.d. mixture prior
```

Comment:
Is this a bit of a weird name for the parameter now? We can change it easily enough, as we haven't made a release with this parameter specified

Reply:
changed to `prior_mixture_dim`